relayer/application_relayer.go

```diff
@@ -125,12 +129,12 @@ func NewApplicationRelayer(
 // Process [msgs] at height [height] by relaying each message to the destination chain.
 // Checkpoints the height with the checkpoint manager when all messages are relayed.
 // ProcessHeight is expected to be called for every block greater than or equal to the [startingHeight] provided in the cosntructor
-func (r *ApplicationRelayer) ProcessHeight(height uint64, messages []messages.MessageHandler) error {
+func (r *ApplicationRelayer) ProcessHeight(height uint64, handlers []messages.MessageHandler) error {
 	var eg errgroup.Group
-	for _, message := range messages {
-		m := message
+	for _, handler := range handlers {
+		h := handler
```

Comment:
Can we leave a comment on this line that this is to capture the loop variable, and can be removed when we move to go `1.22`